src/services/TextRenderService.js

```diff
@@ -30,14 +30,20 @@ export function hideLinks() {
  *
  * @param {int} pageNum
  */
-function createTextLayerDOMIfNotExist(pageNum) {
+function createTextLayerDOMIfNotExist(pageNum, reference = false) {
```

Comment:
What is reference? Missing documentation.

Reply:
This is not good, these should be 2 separeted functions.

Reply:
Done, separated in two functions!